src/explore-education-statistics-common/src/components/form/FormComboBox.tsx

```diff
@@ -156,158 +158,210 @@ const FormComboBox = ({
       onKeyDown={handleContainerInteraction}
       ref={containerRef}
     >
-      <div
-        // eslint-disable-next-line jsx-a11y/role-has-required-aria-props
+      <label
+        className={classNames('govuk-label', classes.inputLabel)}
+        htmlFor={`${id}-input`}
+      >
+        {inputLabel}
+      </label>
+
+      <input
+        type="text"
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...inputProps}
         role="combobox"
-        aria-expanded={renderedOptions ? renderedOptions.length > 0 : false}
-        aria-owns={`${id}-options`}
+        className={classNames(inputStyles.searchInput, 'govuk-input')}
+        id={`${id}-input`}
+        value={value}
+        ref={inputRef}
+        aria-autocomplete="list"
         aria-haspopup="listbox"
-        className="govuk-form-group"
-      >
-        <label
-          className={classNames('govuk-label', classes.inputLabel)}
-          htmlFor={`${id}-input`}
-        >
-          {inputLabel}
-        </label>
-
-        <input
-          type="text"
-          // eslint-disable-next-line react/jsx-props-no-spreading
-          {...inputProps}
-          aria-autocomplete="list"
-          aria-activedescendant={
-            selectedOption > -1 ? `${id}-option-${selectedOption}` : undefined
-          }
-          aria-controls={`${id}-options`}
-          className="govuk-input"
-          id={`${id}-input`}
-          value={value}
-          ref={inputRef}
-          onChange={event => {
-            event.persist();
-
-            setValue(event.target.value);
-            optionRefs.current = {};
-
-            onInputChange(event);
-            toggleShowOptions(true);
-          }}
-          onKeyDown={event => {
-            if (event.key === 'ArrowUp' || event.key === 'ArrowDown') {
-              selectNextOption(event);
+        aria-controls={`${id}-options`}
+        aria-expanded={renderedOptions ? renderedOptions.length > 0 : false}
+        aria-activedescendant={
+          selectedOption > -1 ? `${id}-option-${selectedOption}` : undefined
+        }
+        onChange={event => {
+          event.persist();
 
-              if (listBoxRef.current) {
-                listBoxRef.current.focus();
-              }
-            }
+          setValue(event.target.value);
+          optionRefs.current = {};
 
-            if (event.key === 'Escape') {
-              setValue('');
-              toggleShowOptions(false);
+          onInputChange(event);
+          toggleShowOptions(true);
+        }}
+        onKeyDown={event => {
+          if (event.key === 'ArrowUp' || event.key === 'ArrowDown') {
+            selectNextOption(event);
+
+            if (listBoxRef.current) {
+              listBoxRef.current.focus();
             }
```

Comment:
We don't need this anymore - it'll also interfere with the `useEffect` based way of focusing an option